nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/HealthCheckService.java

```diff
@@ -265,8 +265,11 @@ public UnprocessedChangeEvents changed(PropertyChangeEvent[] events) {
             public <T extends ConfigBeanProxy> NotProcessed changed(TYPE type, Class<T> changedType, T changedInstance) {
                 if(changedType.equals(HealthCheckServiceConfiguration.class)) {
                     configuration = (HealthCheckServiceConfiguration) changedInstance;
+                    return null;
+                }
+                else {
+                    return new NotProcessed("Unimplemented by HealthCheckService");
```

Comment:
I think you should only return a NotProcessed only when you have Vetoed a change. Not when you have ignored a change.